src/main/java/carpet/script/utils/ShapeDispatcher.java

```diff
@@ -625,6 +629,68 @@ public static int particleMesh(List<ServerPlayer> playerList, ParticleOptions pa
         }
     }
 
+    public static class Polyface extends ExpiringShape
+    {
+        @Override
+        public long calcKey(){
+            long hash = super.calcKey();
+            hash ^= 2;                     hash *= 1099511628211L;
```

Comment:
use a unique ID here - 2 is used by a line. not that it would matter.

I need to write a proper builder for this hasher